editor/src/core/tailwind/tailwind.ts

```diff
@@ -96,8 +95,9 @@ function getTailwindConfig(
   if (tailwindFile != null && isTextFile(tailwindFile)) {
     try {
       const requireResult = requireFn('/', TailwindConfigPath)
-      if (requireResult?.default != null) {
-        const twindConfig = convertTailwindToTwindConfig(requireResult.default)
+      const rawConfig = requireResult?.default ?? requireResult
```

Comment:
here you prioritize default over "wildcard", in line 38 it's the other way around.

for reference, this is the babel-emitted function:
```
function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }
```

I think we should actually put this function in a Util file, and use it!

Reply:
Oh wait, I totally forgot, we actually already have this! Will update and use that

Reply:
oh we do? ha!

Reply:
Yeah, we had to use something like that as part of the module loading. This has been fixed and is now using this https://github.com/concrete-utopia/utopia/blob/master/editor/src/core/es-modules/commonjs-interop.ts#L28